girder/girder_large_image/rest/large_image_resource.py

```diff
@@ -449,56 +449,100 @@ def _deleteCachedImages(self, spec, associatedImages=False, imageKey=None):
 
     @access.user(scope=TokenScope.DATA_WRITE)
     @autoDescribeRoute(
-        Description('Create new large images for all items within a folder.')
-        .notes('Does not work for items with multiple files and skips over items with '
-               'existing or unfinished large images.')
-        .modelParam('id', 'The ID of the folder.', model=Folder, level=AccessType.WRITE,
-                    required=True)
-        .param('force', 'Whether creation job(s) should be forced for each large image.',
+        Description('Create large images for all items within a folder.')
+        .notes('Does not work for items with multiple files.')
+        .modelParam('id', 'The ID of the folder.', model=Folder,
+                    level=AccessType.WRITE, required=True)
+        .param('createJobs', 'Whether job(s) should be used to create each '
+               'large image. Select true to use a job if needed, false to '
+               'never use a job, and always to always use a job.',
+               required=False, default='false', dataType='string',
+               enum=['true', 'false', 'always'])
```

Comment:
Based on @manthey 's description in #1139 I get the impression this should be split into two options.
ie 'force' for always and 'localJob' making both boolean as seen here:
https://github.com/girder/large_image/blob/56193ce5784cf60c2591bd570a09aebaa3bef1fa/girder/girder_large_image/rest/tiles.py#L199-L205